evm-template/runtime/src/configs/xcm_config.rs

```diff
@@ -42,6 +51,16 @@ parameter_types! {
     pub BalancesPalletLocation: Location = PalletInstance(<Balances as PalletInfoAccess>::index() as u8).into();
     pub RelayChainOrigin: RuntimeOrigin = cumulus_pallet_xcm::Origin::Relay.into();
     pub UniversalLocation: InteriorLocation = Parachain(ParachainInfo::parachain_id().into()).into();
+        // Self Reserve location, defines the multilocation identifiying the self-reserve currency
+    // This is used to match it also against our Balances pallet when we receive such
+    // a Location: (Self Balances pallet index)
+    // We use the RELATIVE multilocation
+    pub SelfReserve: Location = Location {
+        parents:0,
+        interior: [
+            PalletInstance(<Balances as PalletInfoAccess>::index() as u8)
```

Comment:
I am not an expert on EVM chains but normally the SelfReserve or native asset of the parachain is `{ parents: 0, interior: Here }`

Is this how it is on other EVM chains?

Reply:
We are following the moonbeam approach in general, this code is from their `xcm_config` for the `moonbeam` runtime. 

If you have any strong preference, we would love to learn more and discuss

Reply:
No preference, its more of a convention that the native asset of a parachain has the same location of the parachain. But I would advise sticking close to moonbeam.

Reply:
Ok great! Will resolve this issue then :)